sites/kit.svelte.dev/package.json

```diff
@@ -12,6 +12,7 @@
 	"devDependencies": {
 		"@sveltejs/adapter-auto": "workspace:^",
 		"@sveltejs/adapter-static": "workspace:^",
```

Comment:
Do we still need the auto (or static) adapters here?

Reply:
nope